src/02 - Object types.md

```diff
@@ -405,6 +405,8 @@ The following notation is used in the tables below:
 | M013 | number          | journalaar                    |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                                                                                                                                                                                                                                        |                                                                                                                                                        |
 | M014 | number          | journalsekvensnummer          |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                                                                                                                                                                                                                                        |                                                                                                                                                        |
 | M015 | number          | journalpostnummer             |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                                                                                                                                                                                                                                        |                                                                                                                                                        |
+|      | string          | journalansvarlig              |    x     |      x      |           |  |  x   |  x   |   x   |  x   |               | <p>An identifier of the person responsible for this Journalpost (similar to M307 Saksmappe#saksbehandler)</p><p>&bull; may not be supported by all systems or partially supported</p>                                                                                                                  |                                                                                                                                                        |
+|      | string          | journalansvarligBrukerIdent   |    x     |      x      |           |  |  x   |  x   |   x   |  x   |               | <p>&bull; may not be supported by all systems or partially supported<                                                                                                                                                                                                                                  |                                                                                                                                                        |
```

Comment:
There is a partially closed tag at the end of the comment.

Reply:
I checked this twice as these are the primary issues I get when editing the files with Atom and still got it wrong... :D

Good catch! Fixed.